src/PagerdutyGateway.php

```diff
@@ -76,7 +76,7 @@ public function __construct(Client $client, array $config)
     public function notify($to, $message, array $options = [])
     {
         $options['to'] = $to;
-
+        $params = [];
```

Comment:
We start building an array of parameters to send on the Guzzle request. So we keep appending to that array because we send it as a reference to other functions. Not needed all the time :)